### PR TITLE
New content for Software Package Profile

### PR DESCRIPTION
**Pull request by pcnorth**
Draft pull request because this needs a content review.
Is the content ok?
Is it in the right place?

<sub> Mention [stepsize] in a comment if you'd like to report some technical debt. See examples [here](https://app.stepsize.com/api/demo-pr-redirect).</sub>

### CODE REVIEW

**Comment by kjc-rkvst** [comment]
Hey,
Sorry - I didn't manage to get to this yesterday. Will do it today.

Kel


________________________________
From: Phil North ***@***.***>
Sent: 28 June 2023 15:54
To: rkvst/rkvst-docs ***@***.***>
Cc: Kelvin Cusack ***@***.***>; Review requested ***@***.***>
Subject: [rkvst/rkvst-docs] New content for Software Package Profile (PR #587)


Draft pull request because this needs a content review.
Is the content ok?
Is it in the right place?

[https://user-images.githubusercontent.com/4775299/87437657-e7332b00-c5ee-11ea-958d-589dfb19d72c.png]<https://user-images.githubusercontent.com/4775299/87437657-e7332b00-c5ee-11ea-958d-589dfb19d72c.png> Mention [stepsize] in a comment if you'd like to report some technical debt. See examples here<https://app.stepsize.com/api/demo-pr-redirect>.

________________________________
You can view, comment on, or merge this pull request online at:

  https://github.com/rkvst/rkvst-docs/pull/587

Commit Summary

  *   1db1a10<https://github.com/rkvst/rkvst-docs/pull/587/commits/1db1a10015e26335d59a159dd5448a8ccee52796> New content for Software Package Profile

File Changes

(2 files<https://github.com/rkvst/rkvst-docs/pull/587/files>)

  *   A content/developers/developer-patterns/software-package-profile/PublicCheck.png<https://github.com/rkvst/rkvst-docs/pull/587/files#diff-4976acb047b7d37ac68e037177a146a77b0cfe36ddfdb1d70d29d489ea8c7018> (0)
  *   A content/developers/developer-patterns/software-package-profile/index.md<https://github.com/rkvst/rkvst-docs/pull/587/files#diff-fae5e68875a5c301d5281e0fde1c34cfc7af1357427f1e90f7f50899aba3a560> (256)

Patch Links:

  *   https://github.com/rkvst/rkvst-docs/pull/587.patch
  *   https://github.com/rkvst/rkvst-docs/pull/587.diff

File: content/developers/developer-patterns/software-package-profile/PublicCheck.png
Comment (kjc-rkvst) [inline content/developers/developer-patterns/software-package-profile/PublicCheck.png:1]: I'm on the fence on what screenshot to show. I would prefer the profile to show the "Software package" profile selected, but then the public checkbox gets bumped down a way.

**Review by kjc-rkvst — COMMENTED** [review]
Content is good and I like the placement of it. It is definitely a dev user pattern.

File: content/developers/developer-patterns/software-package-profile/PublicCheck.png
Comment (pcnorth) [reply content/developers/developer-patterns/software-package-profile/PublicCheck.png:1]: I could write a separate article on how to create a software profile asset and put it in "platform-overview" which would go into the detail on actually creating the asset using multiple screenshots. 
This one is to describe the profile attributes as a developer pattern and I only wanted to highlight the public button. 

We are going to have the same content structure for the document profile. One describing the developer pattern and another on using the UI together with YAML and JSON.

**Review by JAG-UK — APPROVED** [review]

**Pull request merged**